helix-core/src/main/java/org/apache/helix/manager/zk/CallbackHandler.java

```diff
@@ -105,7 +106,9 @@
 @PreFetchChangedData(enabled = false)
 public class CallbackHandler implements IZkChildListener, IZkDataListener {
   private static Logger logger = LoggerFactory.getLogger(CallbackHandler.class);
+  private static AtomicLong CB_UID = new AtomicLong();
```

Comment:
`static final`?
And also give it a more comprehensive name, like `CALLBACK_HANDLER_UID` or `NEXT_ID`?

Reply:
changed.